spec/system/posts_index_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  author = User.create(name: 'Tom & Jerry', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Best friends')
+  subject do
+    Post.new(title: 'Physics', text: 'This is not my first post', comments_counter: 10, likes_counter: 10,
+             author_id: author)
+  end
+
+  before { subject.save }
+
+  it 'title should be present' do
```

Comment:
- [ ] @team great work on your tests however you tested the wrong scenarios, kindly check on the project requirement as I have highlighted below for the tests you are supposed to implement for this page
![image](https://user-images.githubusercontent.com/61408860/233322809-57f8198c-3f1d-4ee3-9fe3-35bdfb53baa5.png)